index.html

```diff
@@ -0,0 +1,206 @@
+                <p style="font-size: 50px">Listo para darte un Atorón?</p>
+                <img src="./Cantarito_inicio.png" style="height: 300px;">
+            </div>
+        </section>
+        <section style="padding-top: 100px;">
+            <div style="text-align: center;">
+                <h2 id="products" style="font-size: 40px;">Productos</h2>
+            </div>
+            <div style="display: flex; justify-content: center; text-align: center;">
+                <div style="padding: 10px 50px;">
+                    <h1>Cantaritos</h1>
+                    <img src="./CantaritosSabores.jpg" style="height: 300px;">
+                </div>
+                <div style="padding: 10px 50px;">
+                    <h1>Micheladas</h1>
+                    <img src="./micheladas.jpg" style="height: 300px;">
+                </div>
+            </div>
+        </section>
+        <section id="reseñas" style="padding-top: 100px;">
+            <div style="text-align: center;">
+                <h2 style="font-size: 40px;">Que dicen nuestros clientes</h2>
+            </div>
+            <div style="display: flex; justify-content: center; text-align: center;">
+                <div style="width: 200px; padding: 10px 20px;">
+                    <img src="./hombre.png" alt="" style="height: 200px;">
+                    <h4>Diego</h4>
+                    <p style="height: 100px;">Estan geniales, nuca había probado los cantaritos de sabores</p>
+                </div>
+                <div style="width: 200px; padding: 10px 20px;">
+                    <img src="./hombre.png" alt="" style="height: 200px;">
+                    <h4>Melo</h4>
+                    <p style="height: 100px;">Me encantaron las micheladas</p>
+                </div>
+                <div style="width: 200px; padding: 10px 20px;">
+                    <img src="./hombre.png" alt="" style="height: 200px;">
+                    <h4>Cova</h4>
+                    <p style="height: 100px;">Su servicio es increible</p>
+                </div>
+                <div style="width: 200px; padding: 10px 20px;">
+                    <img src="./hombre.png" alt="" style="height: 200px;">
+                    <h4>Itzel</h4>
+                    <p style="height: 100px;">Sin duda voy voy a pedir nuevamente el servicio</p>
+                </div>
+                <div style="width: 200px; padding: 10px 20px;">
+                    <img src="./hombre.png" alt="" style="height: 200px;">
+                    <h4>Josué</h4>
+                    <p style="height: 100px;">Son las mejores bebidas, las recomiendo mucho</p>
+                </div>
+            </div>
+            <div style=" justify-content: center; text-align: center;">
+                <div>
+                    <label>Porcentaje de satisfaccion del cliente:</label>
+                </div>
+                <div>
+                    <progress value="80" max="100" style="height: 50px;"> 32% </progress>
```

Comment:
```suggestion
                    <progress value="80" max="100" style="height: 50px;"> 80% </progress>
```
Aunque el valor no sea visible en el navegador, para mantener semántica